### ISSUE TITLE
Branch setup after 0.2 release

### ISSUE THREAD

**Issue by duncandewhurst** [GFDRR/rdl-standard#251]
Now that 0.2 is released, I recommend that we delete the `dev` branch and create the following protected branches:

* `0.2-dev` - staging branch for non-normative changes (e.g. documentation and examples)
* `0.3-dev` - staging branch for normative changes (e.g. schema and codelists)

Whilst the remaining non-normative issues on the project board are being addressed, `0.2-dev` can be the default base branch and once work on 0.3 begins we can make `0.3-dev` the default base branch.

@stufraser1 @matamadio let me know what you think.

**Comment by stufraser1** [comment]
Please go ahead

**Comment by duncandewhurst** [comment]
I've created `0.2-dev` and `0.3-dev` from `main`, protected them and set `0.2-dev` as the default branch.

I also created a PR to merge `dev` into `0.2-dev` so that the commits that were made to `dev` after the 0.2 release are not lost. Once that PR is merged, `dev` can be deleted.

**Comment by duncandewhurst** [comment]
I've also deleted old branches that were related to closed PRs.

It would be good to tidy up the remaining branches that do not have an associated PR:

* [`SC_notes_Sep2023`](https://github.com/GFDRR/rdl-standard/tree/SC_notes_Sep2023) - looks like an active branch so we can leave this one
* [`2-addHazardMappings`](https://github.com/GFDRR/rdl-standard/tree/2-addHazardMappings) - superseded by https://github.com/GFDRR/rdl-standard/pull/239 so I think this branch can be deleted. @stufraser1 please confirm
* [`gh-pages`](https://github.com/GFDRR/rdl-standard/tree/gh-pages) - the docs are no longer hosted on GitHub pages so I think this can be deleted. @matamadio @stufraser1 please confirm.

**Comment by stufraser1** [comment]
> * [`SC_notes_Sep2023`](https://github.com/GFDRR/rdl-standard/tree/SC_notes_Sep2023) - please leave, need to finalise those notes and will merge then delete branch.


> * [`2-addHazardMappings`](https://github.com/GFDRR/rdl-standard/tree/2-addHazardMappings) - Agree can be deleted.


> [gh-pages](https://github.com/GFDRR/rdl-standard/tree/gh-pages) - Agree can be deleted.

**Comment by duncandewhurst** [comment]
Great. I've done that.
